src/documents/tests/test_workflows.py

```diff
@@ -0,0 +1,1019 @@
+from datetime import timedelta
+from pathlib import Path
+from unittest import mock
+
+import pytest
+from django.contrib.auth.models import Group
+from django.contrib.auth.models import User
+from django.utils import timezone
+from rest_framework.test import APITestCase
+
+from documents import tasks
+from documents.data_models import ConsumableDocument
+from documents.data_models import DocumentSource
+from documents.matching import document_matches_workflow
+from documents.models import Correspondent
+from documents.models import CustomField
+from documents.models import Document
+from documents.models import DocumentType
+from documents.models import MatchingModel
+from documents.models import StoragePath
+from documents.models import Tag
+from documents.models import Workflow
+from documents.models import WorkflowAction
+from documents.models import WorkflowTrigger
+from documents.signals import document_consumption_finished
+from documents.tests.utils import DirectoriesMixin
+from documents.tests.utils import FileSystemAssertsMixin
+from paperless_mail.models import MailAccount
+from paperless_mail.models import MailRule
+
+
+@pytest.mark.django_db
```

Comment:
Was this necessary?  It's the only instance, and I thought an APITestCase would take care of any database stuff

Reply:
I think that was just in there from before, yea seems fine without, thanks!